server.js

```diff
@@ -1,6 +1,6 @@
 const app = require('./src/app')
-const port = process.env.NODE_PORT || 3000
-const host = process.env.NODE_HOST || '0.0.0.0'
+const port = process.env.PORT || 3000
+const host = process.env.HOST || '0.0.0.0'
```

Comment:
Actually, I'd like to keep support for the `NODE_PORT` and `NODE_HOST`. On some systems, setting `PORT`, or `HOST` env variable might cause some issues :sweat_smile: .

Could you add them as fallback values?

Reply:
I've added aliases for the env variables.